src/doc/rustc/src/codegen-options/index.md

```diff
@@ -62,24 +62,42 @@ the linker.
 
 ## embed-bitcode
 
-This flag controls whether or not the compiler puts LLVM bitcode into generated
-rlibs. It takes one of the following values:
+This flag controls whether or not the compiler embeds LLVM bitcode into object
+files. It takes one of the following values:
 
 * `y`, `yes`, `on`, or no value: put bitcode in rlibs (the default).
 * `n`, `no`, or `off`: omit bitcode from rlibs.
 
-LLVM bitcode is only needed when link-time optimization (LTO) is being
-performed, but it is enabled by default for backwards compatibility reasons.
+The default value of this option is "yes".
```

Comment:
Remove this sentence, it's covered by "(the default)" above, and other boolean options with static defaults don't have a sentence like this.

Reply:
Whoops, missed that!